### PR TITLE
chore(NOJIRA-123): Add hubspot tracking option documentation

### PR DESCRIPTION
**Pull request by eliottparis**
This PR adds the embed configuration documentation's `hubspot` tracking option to the [Available options](https://www.typeform.com/developers/embed/configuration/#available-options).

The `hubspot` option has been introduced by PR #512, but documentation was missing.

Fixes https://github.com/Typeform/embed/issues/651.

### CODE REVIEW

**Review by mathio — APPROVED** [review]
Thank you for the update 🙇

File: packages/embed/README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ Closing and opening a typeform in modal window will restart the progress from th
...
 | [tracking](https://codesandbox.io/s/github/Typeform/embed-demo/tree/main/demo-html/widget-html)                              | object                      | [tracking parameters](https://help.typeform.com/hc/en-us/articles/360050914311-What-is-UTM-tracking-) to be passed to the form in [URL query string](https://developer.mozilla.org/en-US/docs/Web/API/URL/search)                                                                                      | `undefined`                                                   |
+| hubspot                                                                                                                      | boolean                     | enable [HubSpot tracking](https://www.typeform.com/help/a/hub-spot-integration-installation-and-setup-360029578071/)                                                                                                                                                                                   | `false`                                                       |
```
Comment (mathio) [inline packages/embed/README.md:129]: This doc already has `hubspot` in the table.

**Review by mathio — CHANGES_REQUESTED** [review]
Please update the README file.

File: packages/embed/README.md (hunk 1 above)
Comment (eliottparis) [reply packages/embed/README.md]: Oh, sorry I didn't see it, I removed it.

**Comment by eliottparis** [comment]
@mathio Hi, I reverted my changes from `packages/embed/README.md` (I didn't noticed it was already documented here, sorry).
I also rephrased the option description in `docs/configuration.md` to match the one of `packages/embed/README.md` for consistency.

**Review by mathio — APPROVED** [review]

**Pull request merged**